platform/stm32/export/aux/i2c_bus.hpp

```diff
@@ -718,7 +720,38 @@ void i2c_bus<i2c_config>::irq_ev_handler()
 template<class i2c_config>
 void i2c_bus<i2c_config>::irq_er_handler()
 {
-    // TODO add error handling
+    constexpr auto irqn  = pick_er_irqn();
+
+    irq::clear(irqn);
+    constexpr auto i2c = pick_i2c();
+
+    // Explain difference between Events and Flags ?????????
+    // I2C_GetLastEvent() is suitable when multiple flags are monitored at the same time.
+    // Using the function I2C_GetFlagStatus() return the status of one single flag.
```

Comment:
I asked you to explain that `I2C_GetLastEvent()` returns type which is compatible with `I2C_FLAG_*` definitions.